x-pack/legacy/plugins/reporting/server/lib/validate/validate_encryption_key.ts

```diff
@@ -4,17 +4,25 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
 import crypto from 'crypto';
 import { ServerFacade, Logger } from '../../../types';
 
-export function validateConfig(serverFacade: ServerFacade, logger: Logger) {
+export function validateEncryptionKey(serverFacade: ServerFacade, logger: Logger) {
   const config = serverFacade.config();
 
   const encryptionKey = config.get('xpack.reporting.encryptionKey');
   if (encryptionKey == null) {
```

Comment:
Not sure if this was working properly before, but it should be working now with 4c85953

Reply:
I tested 7.5 and this is working properly. Getting a `null` encryptiongKey requires running Kibana from a build, so this block is a bit hard to test